src/language-html/parser-html.js

```diff
@@ -67,6 +67,16 @@ function normalizeAngularControlFlowBlock(node) {
   };
 }
 
+function normalizeAngularICUExpression(node) {
```

Comment:
I prefer lowercase acronyms first in camelcase, but I guess we don't strictly follow any rule about names in code base.

 https://stackoverflow.com/questions/15526107/acronyms-in-camelcase